lib/ansible/modules/cloud/azure/azure_rm_routetable.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Tom Vachon, <github@thomasvachon.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_routetable
+version_added: ""
```

Comment:
This should be: `version_added: "2.5"`

Reply:
Matt, I would but the entire Azure setup doesnt run on 2.5.  Its complaining about the azure python module.  If you have a 2.5 requirements.txt which you know works, I'd be happy to update and test.  I was making unit tests now.